app/src/main/java/com/wednesday/banklocator/model/IfscResponse.kt

```diff
@@ -1,13 +1,15 @@
-package com.wednesday.banklocator.model
-
-data class IfscResponse(
-    val address: String,
-    val bank: String,
-    val branch: String,
-    val city: String,
-    val contact: String,
-    val district: String,
-    var ifsc: String,
-    val rtgs: Boolean,
-    val state: String
-)
+package com.wednesday.banklocator.model
+
+data class IfscResponse(
+    val ADDRESS: String,
```

Comment:
Do not use capital case

Reply:
I tried it with "small letters" but I changed it back as this code wasn't working with "CAPITAL Letters".